### DISCUSSION TITLE
Hitchain scam?

### DISCUSSION THREAD

**Discussion by Frankusky** [community/community#22559] [New to GitHub]
<p>I recently recieved a mail from <a href="http://HitChain.org" rel="nofollow noopener">HitChain.org</a>, which states that for my effort supporting open source will gift me around 1250 <em>hit tokens</em> based on my numbers of commits, issues and pull requests on GitHub.</p>
<p>Since I dont move around <em>block chain</em> world, I dont know if this is scam, both url and page looks ok for me and I dont found any page speak well or badly about hitchain. I left you a copy of my mail:</p>
<hr>
<p>Rewarding letter</p>
<p>Date:July  22，2018</p>
<p>Dear Developer:</p>
<p>   HitChain Foundation would like to reward you as an Open Source enthusiast with some Hit tokens based on your work on Github.<br>
  Here is your achievements:<br>
Commits: 248   PullRequests: 0  Issues: 25</p>
<p>      According to our award rules(this time only),you can apply for 1250 complimentary Hit tokens from this page:</p>
<p>url to <a href="http://hitchain.org" rel="nofollow noopener">hitchain.org</a></p>
<p>      Please click the link above and get your HIT token reward as soon as possible.</p>
<p>    </p>
<p>About HitChain Foundation:</p>
<p>   Hitchain aims to implement a disruptive ‘Hit’ protocol and ecosystem to improve the ‘Git’ source code management platform by leveraging the power of IPFS and blockchain technologies. The goal of Hitchain is to build an autonomous community of the developers，by the developers and for the developers. For more information please visit our website <a href="http://hitchain.org" rel="nofollow noopener">http://hitchain.org</a> or telegram group: <a href="https://t.me/HitChain_EN" rel="nofollow noopener">https://t.me/HitChain_EN</a>.</p>
<p>Hitchain（symbol：HIT）went public at August 3, 2018 on exchange：Huobi Global( <a href="https://www.hbg.com" rel="nofollow noopener">https://www.hbg.com</a> ) and GateIO(<a href="https://gateio.io/" rel="nofollow noopener">https://gateio.io/</a>).<br>
We are look forward of your participation in the Hitchain community.</p>
<p>Let’s keep open sourcing!</p>
<p>HitChain Foundation  </p>

**Comment by Djfe** [comment]
<p>I was mailed as well</p>
<p><img src="https://i.imgur.com/HpvFz7X.png" alt=""></p>

**Comment by Djfe** [comment]
<p>I just noticed that the date at the top of the mail is inaccurate.</p>
<p>I got the mail yesterday. (06.07.2018 15:13 UTC+2)</p>
<p>My guess is phishing</p>

**Comment by Djfe** [comment]
<p>An article I found:</p>
<p><a href="https://hackernoon.com/hitchain-the-community-powered-blockchain-and-newest-edition-to-huobi-global-d0e6461e7928" rel="nofollow noopener">https://hackernoon.com/hitchain-the-community-powered-blockchain-and-newest-edition-to-huobi-global-d0e6461e7928</a></p>
<p>(still haven’t opened either <a href="http://hitchain.io" rel="nofollow noopener">hitchain.io</a> or <a href="http://hitchain.org" rel="nofollow noopener">hitchain.org</a>, which are both mentioned in the mail)</p>

**Comment by Djfe** [comment]
<p>Actually I got 5 mails by them ^^</p>
<p>(first one from 05.08.2018 03:48 UTC+2)</p>

**Comment by Kwasniok** [comment]
<p>Hi Frankusky an Djfe! I would like to extend your research with some links I found (so others can follow this discussion more quickly) and add my own ideas about this topic:</p>
<p>I have received a similar letter from <a href="http://hitchain.io" rel="nofollow noopener">hitchain.io</a> . A quick research revealed that there is a company called <em>HitChain Foundation</em> out there. It is listed as a registered company in Singapore <a href="https://www.sgpbusiness.com/company/Hitchain-Foundation-Limited" rel="nofollow noopener">here</a> and has a public <a href="http://hitchain.org/pdf/HitChain-Whitepaper-en.pdf" rel="nofollow noopener">white paper</a> on their web site which seems to me to be too detailed for an easy scam. I would say if it was a scam it is a rather huge one. But I can not recommend to trust the link (yet) since I could not find a trustworthy statement about this project from a reliable source as well.</p>
<p>Here is what I observed regarding GitHub and HitChain:</p>
<p>HitChain is presented in the white paper (e.g. p.19) as an alternative to GitHub and similar projects. They want <em>" <em><em>to overcome the limitation of Git protocol"</em> via blockchain technology, distributed software etc. and they <em>"</em></em> hope to surpass GitHub as the largest Open-source platform</em>_"_. The HitChain repository was recently (within the last month or so) removed from GitHub. Neither of them advertises a collaboration between them. And I was contacted via my private e-mail address on GitHub which should not be available to them via my public profile page easily. The number of commits, pull requests and issues in the e-mail is correct in my case.</p>
<p>Therefore I think HitChain is real and there is a conflict of interest between both of them and HitChain queried GitHub’s website without their approval as a somewhat aggressive opening move against them. (I do not claim that their methods included any illegal activities!) If this is true I would not trust them due to methods they utilized.</p>
<p>I would like to have an official statement from GitHub since I want to know more about this affair(?) and I am concerned about a potential leakage of my private data.</p>

**Comment by Frankusky** [comment]
<p>Thanks <a class="mention" href="/kwasniok">@kwasniok</a>! I was expecting an official statement from GitHub or HitChain, but there is nothing on network that links them. I would like to add that yesterday I found in this <a href="https://icodrops.com/hitchain/" rel="nofollow noopener">page</a> (now removed), that each hit token has a value of 0.64 USD, that in my case, would be around 800USD, I dont understand well how block chains works, but that its too much money to be gifted in that way, that made me think that it was scam.</p>

**Comment by Frankusky** [comment]
<p>Thanks <a class="mention" href="/djfe">@djfe</a>, but that links just looks like a copy paste of the hitchain website… doesnt provide any special information :confounded:</p>

**Comment by Djfe** [comment]
<p><img src="https://i.imgur.com/qicN1nD.png" alt=""></p>

**Comment by Kwasniok** [comment]
<p>Hey <a class="mention" href="/frankusky">@frankusky</a>! Thank you for adding this crucial information!</p>
<p>I have used GitHub very rarely in the latest years and I got a score of 1110 for 349 commit, 11 issues and less than 20.000 lines of almost worthless code? This is definitively way too much money. One of the claimed investors ranks the value of the HitChainCoin on <a href="https://support.okex.com/hc/en-us/articles/360012283332-HitChainCoin-HIT-" rel="nofollow noopener">this website</a>. (You would get approx. 450USD according to this. Still too much.) Maybe the tokens cannot be traded for fiat money?</p>
<p>And why is the white paper very detailed about the economic part but we get only little information about who we interact with code in this system. E.g. there is no reference to <em>diff files</em> or <em>hash algorithms</em> wich are important technical parts of tools like git.</p>
<p>I clicked on some of the links on their website. They have links to their pages on (facebook, twitter, telegram, git, medium and gitee). Gitee showed me <a href="https://gitee.com/HitChain/HitChain/blob/master/README.md" rel="nofollow noopener">this</a> repository which is live but completely empty? There are wo tweets about their currency going live. Their removed github project. The telegram chat is just filled with people texting in many languages I cannot understand. All in all I see <strong>nothing one could set-up with litte effort</strong> despite setting up a crypto currency which should be easy if you have many investors specialized in this area.</p>
<p>Is this the next generation dot-com bubble? All I know is: They have a listed crypto currency and a business model which they both proudly present but not a single line of code or chat paraphrasing it to show us which tells us how they deal with the programmers input, how its distributed and so on. So at this point in time their focus lies clearly on the monetary aspects.</p>
<p>I have to admit that I do not understand the intensions of HitChain at all.</p>

**Comment by Kwasniok** [comment]
<p>GitHub was or is about to be buyed by Microsoft. (see <a href="http://fortune.com/2018/06/04/microsoft-buy-github/" rel="nofollow noopener">here</a>) Is this an intrigue?</p>

**Comment by Frankusky** [comment]
<p>Yes, was bought by Microsoft <a href="https://www.theverge.com/2018/6/4/17422788/microsoft-github-acquisition-official-deal" rel="nofollow noopener">https://www.theverge.com/2018/6/4/17422788/microsoft-github-acquisition-official-deal</a></p>

**Comment by Kwasniok** [comment, marked as answer]
<p>I am now very confident that it is a <strong>scam</strong>. Just search for <strong>ICO scam</strong> (<a href="https://www.investopedia.com/terms/i/initial-coin-offering-ico.asp" rel="nofollow noopener">initial coin offering</a>) like <a href="https://techcrunch.com/2017/12/07/6-red-flags-of-an-ico-scam/" rel="nofollow noopener">here</a>. They want us to trade their coins for other (proper) crypto currencies.</p>
<p>Try searching the mentioned people on their website. I could not find a second source supporting the fact that they work for HitChain. There is another important detail: Their website does not have a <em>LEGAL</em> or <em>CONTACT</em> button at the bottom of the page. Many countries in the world require a company to have these sections by law.</p>
<p>And there are at least two websites of them with different token distributions:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/065e4420980a8e31ae85d1e511b28a67e7d27c3b.jpeg" data-download-href="/uploads/short-url/UkPoTOZ9heycUidRzVGyw8yF2H.jpg?dl=1" title="Bildschirmfoto 2018-08-08 um 02.26.22.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7516208/181071989-c219b5ab-7d7b-4148-ba6e-e0ec65abbfea.jpg" alt="Bildschirmfoto 2018-08-08 um 02.26.22.jpg" data-base62-sha1="UkPoTOZ9heycUidRzVGyw8yF2H" width="649" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/065e4420980a8e31ae85d1e511b28a67e7d27c3b_2_649x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/0/065e4420980a8e31ae85d1e511b28a67e7d27c3b_2_973x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/0/065e4420980a8e31ae85d1e511b28a67e7d27c3b_2_1298x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/065e4420980a8e31ae85d1e511b28a67e7d27c3b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Bildschirmfoto 2018-08-08 um 02.26.22.jpg</span><span class="informations">3088×2378 237 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/d704b35968bea7cbf03c4051caca32f9524bd358.jpeg" data-download-href="/uploads/short-url/uG8J4Y6yl6LI3SzOnliY1R5Lnao.jpg?dl=1" title="2018-08-08 02.28.10.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7516208/181071990-66dea936-60c6-483d-bcac-64f05b2e9455.jpg" alt="2018-08-08 02.28.10.jpg" data-base62-sha1="uG8J4Y6yl6LI3SzOnliY1R5Lnao" width="281" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/d704b35968bea7cbf03c4051caca32f9524bd358_2_281x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/d/d704b35968bea7cbf03c4051caca32f9524bd358_2_421x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/d/d704b35968bea7cbf03c4051caca32f9524bd358_2_562x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/d704b35968bea7cbf03c4051caca32f9524bd358_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2018-08-08 02.28.10.jpg</span><span class="informations">720×1280 66.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I am sorry. I should have checked these things earlier! Apparently scams got cleverer than I would have ever guessed.</p>
<p>EDIT: Please verify this on your own. (Since I am not an expert.) And if you are convinced it is a scam as well please mention it in this thread.</p>

**Comment by ghost** [comment]
<p>Got a similar mail as well; even though looks confusingly correct, the link to apply for the tokens does NOT link to a hitchain site but to some other  (“<a href="http://subtracker.submail.cn/trilinks?r=aH" rel="nofollow noopener">http://subtracker.submail.cn/trilinks?r=aH</a>…”). Conclusion: scam/phishing.</p>

**Comment by Frankusky** [comment]
<p>Nice research! There is something interesting that its mentioned in the <a href="https://techcrunch.com/2017/12/07/6-red-flags-of-an-ico-scam/" rel="nofollow noopener">second link</a> that you shared:</p>
<blockquote>
<p>If an ICO project is proposing open-source code, an empty or nonexistent GitHub is often a red flag…</p>
</blockquote>
<blockquote>
<p>One of the most obvious red flags for a scam project is the lack of detail on how the technology works. For nontechnical investors, it can be helpful to simply check if a project has any existing files uploaded to public repositories or if a project has a functioning product.</p>
</blockquote>
<p>They don’t have any link to their code (even when it looks like they love open source code), also as mentioned before, the github repository was removed. With all this evidence I am very sure that this is scam.</p>
<p>Thanks everyone for your input! They almost get me :mantongue:</p>

**Comment by nadiajoyce** [comment]
<p>Hi everyone,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Also, I wanted to address the concerns here.</p>
<p>Thanks for surfacing this. GitHub is in no way affiliated with this entity. If you ever receive mail from someone you don’t know and looks suspicious, we always recommend erring on the side of caution. Any official mail from GitHub or announcing new updates to GitHub will be clearly marked as coming from us.</p>
<p>Additionally, we expect our users to comply with our Terms of Service, which prohibits transmitting unsolicited email. We are looking into this further. To prevent messages in the future, you can keep your email address private if you wish by following the steps here:</p>
<p><a href="https://help.github.com/articles/setting-your-commit-email-address-on-github/" rel="nofollow noopener">https://help.github.com/articles/setting-your-commit-email-address-on-github/</a></p>
<p>Please let us know if you have any further questions. Cheers!</p>

**Comment by Kwasniok** [comment]
<p>Hi everyone! Thanks for opening up this thread and your valuable posts! I hope this will help other users as much as it did for me : ) Have a good day and happy programming!</p>

**Comment by Kwasniok** [comment]
<p>Thank you <a class="mention" href="/nadiajoyce">@nadiajoyce</a>! It is exactly what I was looking for! I must have missed this option when I created my account : /</p>

**Comment by mriscoc** [comment]
<p>Hi All,</p>
<p>I have already had my email address settings as private and I had received the same email in my primary address with correct statistics about contributions in Github. How could the scammer get the email address?</p>

**Comment by Kwasniok** [comment]
<p>Hi <a class="mention" href="/mriscoc">@mriscoc</a>,</p>
<p>I belive I have the same issue. When an e-mail address is hidden on github, it is not necessarily hidden in your commits as well. In my case all my repositories are public and some of my commits were signed with my private e-mail address. See the link <a class="mention" href="/nadiajoyce">@nadiajoyce</a> posted above to get instructions on how to use a non-relpy address provided by github as your commit address. It’s easy and affects all future commits.</p>
<p>Apparently changing the email address in commits afterwards seems to be non trivial. One way to fix this is described <a href="https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History" rel="nofollow noopener">here</a> (last section) but it would change <strong>all commits</strong> depending on the commits which contained the old address because their hash values will be different. This might be a no-go if you are working in a team. I haven’t tried this method yet and might test it on a dummy project first.</p>
